.github/CODEOWNERS

```diff
@@ -0,0 +1,9 @@
+# See https://help.github.com/articles/about-codeowners/
+# for more info about CODEOWNERS file
+
+# It uses the same pattern rule for gitignore file
+# https://git-scm.com/docs/gitignore#_pattern_format
+
+**/*                  @sorbet/sorbet-reviewer-roundrobin
```

Comment:
Looking at tbe docs * would work?

Reply:
> Otherwise, Git treats the pattern as a shell glob: "*" matches anything except "/",

>A leading "**" followed by a slash means match in all directories. 

I think no